### ISSUE TITLE
Cannot deploy bind9 when using proxy

### ISSUE THREAD

**Issue by miro-grapeup** [cloudfoundry-community/bind9-boshrelease#14]
Using latest 1.1.3 version with Xenial support fails when proxy is set up. This is caused by this line that tries to install ply:
https://github.com/cloudfoundry-community/bind9-boshrelease/blob/cf84cae1d5fb0b5cc2385a18557279e0449e699f/packages/bind9/packaging#L9

Here is an error we got:
```
Task 41156 | 10:52:55 | Compiling packages: bind9/a2f77814b8953b0e8785ae8d5724e74255f30344 (00:01:10)
                     L Error: Action Failed get_task: Task 76899f7f-0b7f-43f6-6eae-422a53d12125 result: Compiling package bind9: Running packaging script: Running packaging script: Command exited with 1; Stdout:   Cannot fetch index base URL https://pypi.python.org/simple/
  Could not find any downloads that satisfy the requirement ply
Cleaning up...
No distributions at all found for ply
Storing debug log for failure in /root/.pip/pip.log
, Stderr: + export PYTHONPATH=
+ PYTHONPATH=
+ source /var/vcap/packages/python-2.7/bosh/compile.env
++ export PATH=/var/vcap/packages/python-2.7/bin:/var/vcap/bosh/bin:/usr/local/bin:/usr/local/sbin:/bin:/sbin:/usr/bin:/usr/sbin:/usr/X11R6/bin
++ PATH=/var/vcap/packages/python-2.7/bin:/var/vcap/bosh/bin:/usr/local/bin:/usr/local/sbin:/bin:/sbin:/usr/bin:/usr/sbin:/usr/X11R6/bin
++ export PYTHONPATH=/var/vcap/packages/python-2.7/lib/python2.7/site-packages:
++ PYTHONPATH=/var/vcap/packages/python-2.7/lib/python2.7/site-packages:
++ grep -c '^processor' /proc/cpuinfo
+ cpus=2
+ version=9.14.0
+ pip install ply

Task 41156 | 10:54:19 | Error: Action Failed get_task: Task 76899f7f-0b7f-43f6-6eae-422a53d12125 result: Compiling package bind9: Running packaging script: Running packaging script: Command exited with 1; Stdout:   Cannot fetch index base URL https://pypi.python.org/simple/
  Could not find any downloads that satisfy the requirement ply
Cleaning up...
No distributions at all found for ply
Storing debug log for failure in /root/.pip/pip.log
, Stderr: + export PYTHONPATH=
+ PYTHONPATH=
+ source /var/vcap/packages/python-2.7/bosh/compile.env
++ export PATH=/var/vcap/packages/python-2.7/bin:/var/vcap/bosh/bin:/usr/local/bin:/usr/local/sbin:/bin:/sbin:/usr/bin:/usr/sbin:/usr/X11R6/bin
++ PATH=/var/vcap/packages/python-2.7/bin:/var/vcap/bosh/bin:/usr/local/bin:/usr/local/sbin:/bin:/sbin:/usr/bin:/usr/sbin:/usr/X11R6/bin
++ export PYTHONPATH=/var/vcap/packages/python-2.7/lib/python2.7/site-packages:
++ PYTHONPATH=/var/vcap/packages/python-2.7/lib/python2.7/site-packages:
++ grep -c '^processor' /proc/cpuinfo
+ cpus=2
+ version=9.14.0
+ pip install ply


Task 41156 Started  Tue Jul 23 10:48:15 UTC 2019
Task 41156 Finished Tue Jul 23 10:54:19 UTC 2019
Task 41156 Duration 00:06:04
Task 41156 error

Updating deployment:
  Expected task '41156' to succeed but state is 'error'

Exit code 1
```

**Comment by jhunt** [comment]
Ew..  I wonder if we even _need_ ply (or python for that matter).

**Comment by jhunt** [comment]
I removed the Python dependencies and tested with 9.14.0 (in 93a8389) and it works great.  You can either test with a dev release, or wait until I can get 1.2.0 out, which will hopefully happen later today / this week.

**Comment by jhunt** [comment]
This should be fixed in v1.1.4: https://github.com/cloudfoundry-community/bind9-boshrelease/releases/tag/v1.1.4
